examples/browser-api-playground/src/useConfigStore.ts

```diff
@@ -24,16 +24,18 @@ const shouldReload = (prevValue: ConfigType, newValue: ConfigType) => {
 const useConfigStore = () => {
     const [config, setConfig] = useState<ConfigType>(() => {
         const value = window.sessionStorage.getItem(storeKey)
-        return value ? merge({}, defaultConfig, JSON.parse(value)) : defaultConfig
+        if (!value) return defaultConfig
+        return JSON.parse(value)
     });
 
     const setPersistentConfig = useCallback((newValue: ConfigType) => setConfig((prevValue: ConfigType) => {
-        window.sessionStorage.setItem(storeKey, JSON.stringify(newValue));
+        const updatedConfig = merge({}, config, newValue)
```

Comment:
Why are we updating this behaviour? With this, I guess we cant unset a key in config. For example:

```
config:

{
   name: 'Naveen'
}

newValue

{
    name: undefined
}

```
updatedConfig will be `{ name: 'Naveen'}`

Reply:
Additionally, we should pass config as a dependancy to `useCallback`

Reply:
>Additionally, we should pass config as a dependancy to useCallback

Oops, used to linter auto adding. Thanks will add

>Why are we updating this behaviour? With this, I guess we cant unset a key in config

Oh, assumed merge is going to override with source, will fix this. 

But just to confirm, currently we are able to unset only the fields that are `undefined` in the `defaultConfig` and set using `window.setConfig`. We wouldn't be able to unset any field that is not undefined in `defaultConfig`, is this correct?

```
const value = window.sessionStorage.getItem(storeKey)
return value ? merge({}, defaultConfig, JSON.parse(value)) : defaultConfig
```

Reply:
> We wouldn't be able to unset any field that is not undefined in defaultConfig, is this correct?

I suppose there won't be a use case for unsetting fields in `defaultConfig` because the values defined there represent the default behavior for the Embedded widget. By this, I mean that using `defaultConfig` and an empty config would result in the same behavior.